ERCS/erc-alt_gas_fees.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Alternative Gas Fees Capability
+description: A capability allowing wallets to indicate that they have alternative means for paying gas fees.
+author: TODO
+discussions-to: <URL>
```

Comment:
Please create a discussions topic in Eth Magicians with a link to this PR
https://ethereum-magicians.org/c/ercs/57